src/shape/line.js

```diff
@@ -662,5 +663,10 @@ extend(ChartInternal.prototype, {
 
 	isWithinStep(that, y) {
 		return Math.abs(y - d3Mouse(that)[1]) < 30;
-	}
+	},
+
+	shouldDrawPointsForLine(d) {
+		return this.config.line_points === true ||
+			(Array.isArray(this.config.line_points) && this.config.line_points.includes(d.id));
```

Comment:
`Array.prototype.includes` can't be used on IE, unless adding some transpile module like [babel-polyfill](https://babeljs.io/docs/en/babel-polyfill/).
`Array.prototype.indexOf` has more broad compatibility than the `.includes`. I'll take changing that.